ibexa/commerce/4.0.x-dev/manifest.json

```diff
@@ -106,7 +106,10 @@
     "Ibexa\\Bundle\\ImageEditor\\IbexaImageEditorBundle": ["all"],
     "Ibexa\\Bundle\\OAuth2Client\\IbexaOAuth2ClientBundle": ["all"],
     "Ibexa\\Bundle\\ProductCatalog\\IbexaProductCatalogBundle": ["all"],
-    "Ibexa\\Bundle\\CompatibilityLayer\\IbexaCompatibilityLayerBundle": ["all"]
+    "Ibexa\\Bundle\\CompatibilityLayer\\IbexaCompatibilityLayerBundle": ["all"],
+    "Ibexa\\Bundle\\Taxonomy\\IbexaTaxonomyBundle": ["all"],
```

Comment:
Could we keep these above `IbexaCompatibilityLayerBundle`? It should not matter, as yours bundle are fully rebranded but I think it would be cleaner that way.

Reply:
Fixed and force pushed